README.md

```diff
@@ -651,9 +664,9 @@ Let's create a simple app that integrates PSPDFKit and uses the react-native-psp
       }
   });
  ```
-17. Now run the application on the command line: `react-native run-windows`.
-18. Press Yes when Power shell wants to run.
-19. Type 'y' when asking if you want to install the certificate.
+22. Now run the application on the command line: `react-native run-windows`.
+23. Press Yes when Power shell wants to run.
```

Comment:
"Press Yes when Power shell wants to run." -> "Press `Yes` when PowerShell wants to run."